woocommerce/woocommerce-ios
# Custom help center content for "Enter Store Address" screen

Part of: #7547 
- [ ] ⚠️ Please review https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/673 ⚠️
- [ ] ⚠️ @selanthiraiyan update Podfile after https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/673 is merged and released ⚠️

### Description
As a part of improving the help center content this PR adds custom content with FAQs and Answers for "Enter Your Store Address" screen at the following URL.
https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-address

Internal ref - pe5sF9-sZ-p2

### Testing instructions
1. Install and launch the app.
2. Tap `Skip` to reach the login prologue screen.
3. Tap `Enter Your Store Address` button.
4. Tap `Help` -> `Help Center`
5. Observe that you are directed to https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-address in a web view
6. In Xcode debug console observe that the following event is tracked 
```
Tracked support_help_center_viewed, properties: [AnyHashable("source_step"): "start", AnyHashable("help_content_url"): "https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-address", AnyHashable("source_flow"): "login_site_address"]
```
7. Dismiss the web view and navigate to any screen other than "Enter Store Address" screen. (Enter WPCOM email address for example)
8. Tap `Help` -> `Help Center`
9. In Xcode debug console observe that the following event is tracked
```
Tracked support_help_center_viewed
```
10. Observe that the https://woocommerce.com/document/woocommerce-ios/ URL is opened in webview

### Screenshots

https://user-images.githubusercontent.com/524475/186145764-1b42eab9-1c76-4db1-b3e8-824abb82db3d.mov


---
